src/form/form.tsx

```diff
@@ -342,6 +344,12 @@ class Form<MODEL extends object> extends React.Component<Props<MODEL>, State<MOD
         return this.setStateAsync({ isValid });
     }
 
+    private onUpdate() {
```

Comment:
Basically i need the contents of the form at any given time so that the user can actually use them to test if they work or not. In our case the form - mainly - deals with connection profile data - i.e. ftp account - and one of the requirements is to test if the given values can be used to establish a connection.

Reply:
So is it a validation Problem you want to solve?

Reply:
Not really. The values themselves might be perfectly valid - based on the validation criteria provided - but i need to make an API call with some of the data to test if the connection succeeds.

Reply:
The connection test might or might not be successful and that should by no means disable the user from submitting the values. After all he should be able to test the values at any given point in time.

Reply:
Maybe we can challenge the disabling. but you use case should be solvable with validation:
![image](https://user-images.githubusercontent.com/14793495/97855035-8002c900-1cfa-11eb-92e0-dff6130f9a33.png)

Reply:
Can't see how that would solve the requirement I need to fulfil.